srcpkgs/boost/template

```diff
@@ -37,9 +32,17 @@ do_build() {
 	LIBS="$LDFLAGS" ${wrksrc}/bjam -f build.jam --toolset=cc --toolset-root= -d+2 clean
 	LIBS="$LDFLAGS" ${wrksrc}/bjam -f build.jam --toolset=cc --toolset-root= -d+2
 	)
+	
+	cat > user-config.jam <<-__EOF
+	using gcc : ${_toolset#gcc-} : ${CXX} : <cxxflags>"${CXXFLAGS}" <linkflags>"${LDFLAGS}" ;
+	using python : 2.7 : ${XBPS_CROSS_BASE}/usr/bin/python : ${XBPS_CROSS_BASE}/usr/include/python2.7 : ${XBPS_CROSS_BASE}/usr/lib/python2.7 ;
+	using python : 3.6 : ${XBPS_CROSS_BASE}/usr/bin/python3 : ${XBPS_CROSS_BASE}/usr/include/python3.6m : ${XBPS_CROSS_BASE}/usr/lib/python3.6 ;
```

Comment:
Please do not hardcode the Python 3.x version.
Better to open a separate PR for `boost-python3`.

Reply:
Ok I split it up into it's own PR.